src/main/java/calculator/helper/DelimiterHelper.java

```diff
@@ -0,0 +1,31 @@
+package calculator.helper;
+
+import java.util.regex.Pattern;
+
+import calculator.constant.DrawConstant;
+import calculator.exception.CalculatorException;
+import calculator.exception.message.ExceptionMessage;
+
+public class DelimiterHelper {
+	public String extractCustomDelimiter(String formula) {
+		validateCustomDelimiterFormat(formula);
+		return String.valueOf(formula.charAt(DrawConstant.CUSTOM_DELIMITER_INDEX));
+	}
+
+	private static void validateCustomDelimiterFormula(String formula) {
+		if (!(Pattern.matches(DrawConstant.CUSTOM_DELIMITER_FORMULA_REGEX, formula))) {
+			throw new CalculatorException(ExceptionMessage.INVALID_FORMAT_ERROR);
+		}
+	}
```

Comment:
필요없는 괄호는 제거하셔도 좋을 것 같아요!

```suggestion
	private static void validateCustomDelimiterFormula(String formula) {
		if (!Pattern.matches(DrawConstant.CUSTOM_DELIMITER_FORMULA_REGEX, formula)) {
			throw new CalculatorException(ExceptionMessage.INVALID_FORMAT_ERROR);
		}
	}
```